functions/modules/twitterDb.js

```diff
@@ -0,0 +1,60 @@
+// const config = require('../config/config');
+
+var admin = require('firebase-admin');
+
+const TWITTER_HANLDES_DB = 'twitterHandlesAddresses';
+
+const isTwitterHandleRegistered = async handle => {
+  let docRef = admin
```

Comment:
Small thing, but we should basically always use const instead of let unless you really do need to reassign the variable.